Mathlib/Topology/IsLocalHomeomorph.lean

```diff
@@ -211,19 +211,10 @@ theorem isOpenEmbedding_of_injective (hf : IsLocalHomeomorph f) (hi : f.Injectiv
 @[deprecated (since := "2024-10-18")]
 alias openEmbedding_of_injective := isOpenEmbedding_of_injective
 
-/-- A surjective embedding is a homeomorphism. -/
-noncomputable def _root_.Topology.IsEmbedding.toHomeomorph_of_surjective (hf : IsEmbedding f)
-    (hsurj : Function.Surjective f) : X ≃ₜ Y :=
-  Homeomorph.homeomorphOfContinuousOpen (Equiv.ofBijective f ⟨hf.injective, hsurj⟩)
-    hf.continuous (hf.isOpenEmbedding_of_surjective hsurj).isOpenMap
-
-@[deprecated (since := "2024-10-26")]
```

Comment:
Did you accidentally lose this alias?

Reply:
I don't think so, I just moved it together with the declaration. But GitHub is displaying some stuff strangely.

Reply:
Thanks; the alias is definitely still present.